pkg/dssproto/dss.proto

```diff
@@ -76,6 +76,14 @@ message CreateIdentificationServiceAreaParameters {
   string flights_url = 2;
 }
 
```

Comment:
FYI: A lot of the top level proto definitions are unused, ie: verticalAccuracy. Just wanted to highlight, you're call on how you want to address that